docs/odata/v2vsv4.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: V2 vs. V4
+name: vs2vsv4
+parent: OData
+nav_order: 2
+---
+
+## Major Differences
+
+These are the major differences between OData V4 and OData V4:
+
+### Metadata control
+
+In OData V4, the JSON data format now allows to control the amount of metadata that is returned in query responses. There are three levels of metadata supported [(see details)](http://docs.oasis-open.org/odata/odata-json-format/v4.0/os/odata-json-format-v4.0-os.html#_Toc372793040):
+* `full`: The response contains all the metadata needed to describe the response.
+* `minimal`: The response metadata references the metadata document. Information in the metadata document is not repeated in the response.
+* `none`: The response contains no metadata. The application must understand the response structure.
+
+### Search capability
+
+OData V4 adds a new flexible search capability, `$search`. The search feature allows you to query a collection for entities that match a specified search expression. Unlike the existing filter capability, which allows a query to specify that a specific property or properties match certain criteria, the search feature can apply the search expression to any of the properties of an entity.
+
+### Enhanced expand
+
+The `$expand` system query option has been enhanced in OData V4. This feature specifies the related resources to be included in line with retrieved resources. In OData 2, if a single value navigation property is expanded, you get all the properties of the entity if it was a single value navigation property. And if a collection navigation property is expanded, you get all of the entities in the collection and all of the properties of those entities. In OData V4, you can now refine the results using the `$select`, `*`, `$filter` and `$top` operations.
+
+Example: Get all teams that have at least one employee who is older than 42
+
+```js
```

Comment:
i know i'm close to knit picking here, but i'd really like to see any absolute namespace refs in ui5 apps gone so the code is prepped for async op and proper build.
(attention, coded free-hand)
```js
sap.ui.require([
  "sap.ui.model.Filter",
  "sap.ui.model.FilterOperator"
], (Filter, FilterOperator) => {
    // ...
    oTeamsBinding.filter(
        new Filter({
            path : "EMPLOYEES",
            operator : FilterOperator.Any,
            variable : "employee",
            condition : new Filter("employee/AGE", FilterOperator.GT, 42)
        })
    )
})
```

Reply:
Thanks for the feedback!
For OData media handling, I didnt find any major difference to V2. (See: http://docs.oasis-open.org/odata/odata/v4.0/errata03/os/complete/part1-protocol/odata-v4.0-errata03-os-part1-protocol-complete.html#_Toc453752285)
I removed the sentence about SmartControls until we are sure that this is correct.
All other comments are implemented.